ews/src/types/mark_all_read.rs

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use ews_proc_macros::operation_response;
+use serde::Deserialize;
+use xml_struct::XmlSerialize;
+
+use crate::{BaseFolderId, MESSAGES_NS_URI};
+
+/// A request to mark all items in a collection of folders as read.
+///
+/// See <https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/markallitemsasread>
+#[derive(Clone, Debug, XmlSerialize)]
+#[xml_struct(default_ns = MESSAGES_NS_URI)]
+#[operation_response(MarkAllItemsAsReadResponseMessage)]
+pub struct MarkAllItemsAsRead {
+    #[xml_struct(ns_prefix = "m")]
+    pub read_flag: bool,
+    #[xml_struct(ns_prefix = "m")]
+    pub suppress_read_receipts: bool,
+    #[xml_struct(ns_prefix = "m")]
```

Comment:
```suggestion
    pub read_flag: bool,
    pub suppress_read_receipts: bool,
```

The default namespace means we don't use a prefix (`m` isn't defined anywhere).